src/deserialize_sequence.cc

```diff
@@ -319,4 +321,23 @@ void load_sequence_parameters(const std::filesystem::path& folder, Sequence& seq
     sequence.set_step_setup_script(step_setup_script);
 }
 
+std::vector<Tag> parse_tags(gul14::string_view str)
+{
+    const auto tokens = gul14::tokenize_sv(str);
+
+    std::vector<Tag> tags;
+    tags.reserve(tokens.size());
+
+    for (auto tok : tokens)
+    {
+        auto maybe_tag = Tag::from_string(tok);
+
+        // Silently drop invalid tags
```

Comment:
Hmm, in the other cases where a saved Sequence has a bug in the file we throw and do not ignore silently.

e.g. `throw Error("Step must have type declaration");`

Compare my comment about 'now we can not see the reason for a problem' anymore in `Tag`.

Reply:
Let's see. For the timeout we throw if the format is wrong or if the number cannot be parsed:
```cpp
throw Error(gul14::cat("timeout: unable to parse number ('", timeout, "')"));
```
... so that eventually `SequenceManager::load_sequence()` throws.

I guess that behavior makes sense. If something in sequence.lua cannot be parsed, it is better to get an error message than to silently discard information. Right? I would drop the `from_string()` then and simply let the constructor throw.

Reply:
I have implemented the throwing behavior in the last commit. `Tag::from_string()` is now unnecessary and therefore removed.